pages/billy-wilder-theater.vue

```diff
@@ -113,11 +130,35 @@ useHead({
   >
     <div class="one-column">
       <ResponsiveImage
-        v-if="parsedImage.length === 1"
-        :media="parsedImage[0]"
+        v-show="parsedImage && parsedImage.length === 1 && parsedImage[0]?.image && parsedImage[0]?.image?.length === 1"
+        :media="parsedImage[0]?.image?.[0]"
         :aspect-ratio="43.103"
         data-test="hero-image"
-      />
+      >
+        <template
+          v-if="parsedImage?.[0]?.creditText"
+          #credit
+        >
+          {{ parsedImage?.[0]?.creditText }}
+        </template>
+      </ResponsiveImage>
+      <div
+        v-show="parsedCarouselData && parsedCarouselData.length > 1"
```

Comment:
@farosFreed will both hero image and image carousel data show? if both present? or if hero image and image carousel dont exixts will hero image always show on the v-show sections